Jenkinsfile

```diff
@@ -99,7 +99,7 @@ pipeline {
                defaultValue: getPriority(),
                description: 'Priority of this build.  DO NOT USE WITHOUT PERMISSION.')
         string(name: 'TestTag',
-               defaultValue: 'full_regression',
+               defaultValue: '',
```

Comment:
What allows this to be changed?

Reply:
Every stage in this file specifies `default_tags`, so `params.TestTag` is redundant. And I think it might be part of why your previous PR accidentally ran `full_regression` even though you used `Test-tag:`, considering the logic here:
https://github.com/daos-stack/pipeline-lib/blob/50168a6abc6a10fc8a9cd0396eb6526c3d1e02f0/vars/getFunctionalTags.groovy#L20-L36